dev/breeze/src/airflow_breeze/commands/minor_release_command.py

```diff
@@ -0,0 +1,183 @@
+
+
+def commit_changes(version_branch):
+    if CI:
+        console_print("Skipping committing changes in CI")
+        return
+    if confirm_action("Commit the above changes?"):
+        run_command(["git", "add", "-p", "."])
+        run_command(["git", "commit", "-m", f"Update default branches for {version_branch}"])
+
+
+def create_stable_branch(version_branch):
+    if CI:
+        console_print("Skipping creating stable branch in CI")
+        return
+    if confirm_action(f"Create stable branch: v{version_branch}-stable?"):
+        run_command(["git", "checkout", "-b", f"v{version_branch}-stable"])
+        console_print(f"Created branch: v{version_branch}-stable")
+    else:
+        run_command(["git", "checkout", f"v{version_branch}-stable"])
+
+
+def push_test_and_stable_branch(version_branch):
+    if CI:
+        console_print("Skipping pushing test and stable branches in CI")
+        return
+    if confirm_action("Push test and stable branches?"):
+        run_command(["git", "checkout", f"v{version_branch}-test"])
+        run_command(["git", "push", "--set-upstream", "origin", f"v{version_branch}-test"])
+        run_command(["git", "checkout", f"v{version_branch}-stable"])
+        run_command(["git", "push", "--set-upstream", "origin", f"v{version_branch}-stable"])
+
+
+def checkout_main():
+    if confirm_action("We now need to checkout main. Continue?"):
+        run_command(["git", "checkout", "main"])
+        run_command(["git", "pull"])
+
+
+def instruction_update_version_branch(version_branch):
+    if confirm_action("Now, we need to manually update the version branches in main. Continue?"):
+        console_print()
+        console_print(
+            f"Add v{version_branch}-stable and v{version_branch}-test branches "
+            "in codecov.yml (there are 2 places in the file!)"
+        )
+        console_print("Areas to add the branches will look like this:")
+        console_print(
+            """
+            branches:
+                - main
+                - v2-0-stable
+                - v2-0-test
+                - v2-1-stable
+                - v2-1-test
+                - v2-2-stable
+                - v2-2-test
+            """
+        )
+        console_print()
+        console_print(f"Add v{version_branch}-stable to .asf.yaml ({version_branch} is your new branch)")
+        console_print(
+            f"""
+            protected_branches:
+            main:
+                required_pull_request_reviews:
+                required_approving_review_count: 1
+            ...
+            v{version_branch}-stable:
+                required_pull_request_reviews:
+                required_approving_review_count: 1
+            """
+        )
+        console_print("Once you finish with the above. Commit the changes and make a PR against main")
+        confirm_action("I'm done with the changes. Continue?", abort=True)
+
+
+def create_constraints(version_branch):
+    if CI:
```

Comment:
This?

Reply:
I think we could simply skip pushing the changes in CI.

Reply:
Oh, yes, good eye. There are probably more of this class where we are skipping the whole thing vs the portion we can't do.